.travis.yml

```diff
@@ -9,14 +9,14 @@ services:
 - docker
 bundler_args: "--without integration tools maintenance deploy"
 before_install:
-- gem install bundler
-- gem update --system 2.4.5
+- gem update --system
+- gem update bundler
 - gem --version
 matrix:
   include:
-  - rvm: 1.9.3
-  - rvm: 2.1.8
-  - rvm: 2.2.5
+  - rvm: 2.1.10
+  - rvm: 2.2.6
+  - rvm: 2.3.3
```

Comment:
instead of adding `2.3.3` here, what do think about changing all the `2.3.1` runs below to this release instead?